src/ocamlorg_data/ood.mli

```diff
@@ -86,12 +86,12 @@ module Event : sig
   type t =
     { title : string
     ; slug : string
-    ; description : string
+    ; description : string option
+    ; organiser : string option
     ; url : string
     ; date : string
     ; tags : string list
     ; online : bool
-    ; textual_location : string option
```

Comment:
We may not have the data for it now, but from what I can tell it still makes sense to have a textual location, no?